osu.Game/OsuGame.cs

```diff
@@ -1018,6 +1018,8 @@ protected override void UpdateAfterChildren()
 
             var horizontalOffset = 0f;
 
+            // Calculate the horizontal offset using Content, as it gets nested inside a ScalingMode.Everything container
+            // which should apply to overlays, but not get affected by modes like ScalingMode.ExcludeOverlays which shouldn't.
```

Comment:
```suggestion
            // Content.ToLocalSpace() is used instead of this.ToLocalSpace() to correctly calculate the offset with  scaling modes active.
            // Content is a child of a scaling container with ScalingMode.Everything set, while the game itself is never scaled.
            // this avoids a visible jump in the positioning of the screen offset container.
```